File: exo3/exo3.py
class exo3:
    @staticmethod
    def find(mot, lettre):
        count = 0
        for i in range(0, len(mot)):
            if (mot[i] == lettre):
                count += 1
        return count
    
    @staticmethod
    def main():
        mot = raw_input("Veuillez entrer un mot : ")
        lettre = raw_input("Veuillez entrer la lettre a compter :  ")

        print "La lettre {0} est presente {1} fois dans le mot {2}".format(lettre,exo3.find(mot, lettre),mot)
    
exo3.main()

File: exo3/exo3.rb
class Exo3
    def self.find(mot, lettre)
        count = 0
        for i in 0..mot.length
            if (mot[i] == lettre)
                count += 1
            end
        end
        return count
    end
    
    def self.main()
        
        print "Veuillez entrer un mot : "
        mot = gets.chomp
        
        print "Veuillez entrer la lettre a compter : "
        lettre = gets.chomp

        print "La lettre " + lettre + " est presente " + Exo3.find(mot, lettre).to_s + " fois dans le mot " + mot + "\n"
    end
end

Exo3.main()